File: src/integTest/resources/felixProjects/hello/hello/src/main/java/de/thomasvolk/felix/hello/Activator.java
package de.thomasvolk.felix.hello;

import org.osgi.framework.*;

public class Activator implements BundleActivator {

  public void start(BundleContext ctx) {
    System.out.println(this.getClass().getName() + ": start");
  }

  public void stop(BundleContext ctx) {
    System.out.println(this.getClass().getName() + ": stop");
  }
}

File: src/integTest/resources/felixProjects/servlet/helloService/src/main/scala/de/thomasvolk/felix/helloservice/Activator.scala
package de.thomasvolk.felix.helloservice

import org.osgi.framework.{BundleActivator, BundleContext}

class Activator extends BundleActivator {

  def start(ctx: BundleContext) {
    println(this.getClass.getName + ": start")
  }

  def stop(ctx: BundleContext) {
    println(this.getClass.getName + ": stop")
  }
}